codes/Code5.java

```diff
@@ -0,0 +1,25 @@
+package codes;
+public class Code5 extends Thread {
+    public static void main(String[] args) {
+        Thread thread1= new Thread(new Code5());
+        Thread thread2= new Thread(new Code5());
+        thread1.setName("POLO-1");
+        thread1.setPriority(10);
```

Comment:
thread1.setPriority(Thread.MAX_PRIORITY);
```suggestion
        thread1.setPriority(10);
```